File: templates/java-webmvc/subsystem/user/component/src/test/java/{{PACKAGE_NAME}}/user/MockData.java
package {{PACKAGE_NAME}}.user;

import {{PACKAGE_NAME}}.post.model.Post;
import {{PACKAGE_NAME}}.user.model.User;

import java.util.List;

public class MockData {

  static final List<User> users = List.of(
    new User(1, "Talia", "Fritsch"),
    new User(2, "Daisy", "Lubowitz"),
    new User(3, "Chaz", "Rolfson")
  );

  static final List<Post> posts = List.of(
    new Post(1, 1, "Lorem ipsum dolor sit amet.", "Curabitur ultrices lacus lorem, vitae tincidunt lorem convallis."),
    new Post(2, 1, "Maecenas maximus sodales nibh vel.", "Pellentesque scelerisque tortor nec lorem volutpat, sed semper."),
    new Post(3, 3, "Sed tempus, justo at viverra.", "Cras efficitur, nunc eu maximus congue, magna quam.")
  );

}

File: templates/kotlin-webflux/subsystem/user/component/src/test/kotlin/{{PACKAGE_NAME}}/user/MockData.kt
package {{PACKAGE_NAME}}.user

import {{PACKAGE_NAME}}.post.model.Post
import {{PACKAGE_NAME}}.user.model.User

object MockData {
  val users = listOf(
    User(id = 1, familyName = "Talia", givenName = "Fritsch"),
    User(id = 2, familyName = "Daisy", givenName = "Lubowitz"),
    User(id = 3, familyName = "Chaz", givenName = "Rolfson")
  )

  val posts = listOf(
    Post(id = 1, userId = 1, title = "Lorem ipsum dolor sit amet.", body = "Curabitur ultrices lacus lorem, vitae tincidunt lorem convallis."),
    Post(id = 2, userId = 1, title = "Maecenas maximus sodales nibh vel.", body = "Pellentesque scelerisque tortor nec lorem volutpat, sed semper."),
    Post(id = 3, userId = 3, title = "Sed tempus, justo at viverra.", body = "Cras efficitur, nunc eu maximus congue, magna quam.")
  )
}